### PR TITLE
fix minor bugs

### PR DESCRIPTION
**Pull request by sw-yx**
fixes added:

1. `npm tauri dev` to dev the tauri version otherwise whole thing doesnt work
2. added more native menu items (but somehow keyboard shortcut paste doesnt work)
3. passed in directory string for other computers not named zagreus
4. `text-black` font color for filling in text

### CODE REVIEW

File: src-tauri/src/main.rs [hunk 1]
```diff
@@ -38,9 +41,9 @@ fn main() {
 }
 
 #[tauri::command]
-async fn stable_diffusion_command(command: String) -> String {
+async fn stable_diffusion_command(directory: String, command: String) -> String {
     // let stable_diffusion_directory = Path::new("/Users/zagreus/code/ml/stable-diffusion");
-    let stable_diffusion_directory = "/Users/zagreus/code/ml/stable-diffusion";
+    let stable_diffusion_directory = directory; // "/Users/zagreus/code/ml/stable-diffusion";
```
Comment (breadthe) [inline src-tauri/src/main.rs:46]: This is just embarrassing. I totally didn't mean to leave this hardcoded 😛

**Comment by breadthe** [comment]
Excellent! You fixed the things I mixed 🙇‍♂️ 
I'm taking all of it, but I need to incorporate the stuff from `App.svelte` into a new component that I extracted.

**Comment by breadthe** [comment]
I've incorporated these changes. Thanks a bunch! Will create a new release soon.

**Comment by sw-yx** [comment]
sounds good :)

**Pull request closed without merging**